packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -253,21 +282,48 @@ export function Table<T>(props: TableProps<T>): ReactElement {
         )
     ) : null;
 
+    const cssGridStyles = useMemo(() => {
+        const columnSizes = visibleColumns
+            .map(c => {
+                switch (c.width) {
+                    case "autoFit":
+                        /**
+                         * If auto fit column is the only column, it should fit the space
+                         */
+                        if (visibleColumns.length === 1) {
+                            return "1fr";
+                        }
+                        return "fit-content(100%)";
+                    case "manual":
+                        return `${c.weight}fr`;
+                    default:
+                        return "1fr";
+                }
+            })
+            .join(" ");
+        return {
+            "grid-template-columns": columnSizes + (props.columnsHidable ? ` fit-content(100px)` : "")
+        } as any;
```

Comment:
please avoid `any`

Reply:
Big mistake here, changed to `CSSProperties`